client/lib/react-helpers/index.js

```diff
@@ -9,15 +9,25 @@ import { Provider as ReduxProvider } from 'react-redux';
  * Internal dependencies
  */
 import analytics from 'lib/analytics';
+import Stylizer, { insertCss } from './stylizer';
 
 export function concatTitle( ...parts ) {
 	return parts.join( ' › ' );
 }
 
-export function renderPage( context, component ) {
-	renderWithReduxStore(
-		component,
-		document.getElementById( 'primary' ),
+export function renderPage( component, domContainer, context ) {
```

Comment:
I might wait til you do this until we run the e2e tests against it. I'm on weekend now (Australia time) but @hoverduck should be able to help out Friday US time